File: src/main/java/me/travis/wurstplusthree/util/WhitelistUtil.java
package me.travis.wurstplusthree.util;

import net.minecraft.block.Block;

/**
 * @author Madmegsox1
 * @since 08/05/2021
 */

public class WhitelistUtil {
    public static Block findBlock(String name){
        return Block.getBlockFromName(name);
    }
}

File: src/main/java/me/travis/wurstplusthree/util/WhitelistUtil.kt
package me.travis.wurstplusthree.util

import net.minecraft.block.Block

/**
 * @author Madmegsox1
 * @since 08/05/2021
 */


class WhitelistUtil {
    fun findBlock(name: String): Block {
        return Block.getBlockFromName(name)!!
    }
}